github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
OK so that *sort of* works, but saving the whole cache takes a *looooong* time. It's been sitting at "Generated log placed in..." for over two minute now, as it needs to serialise and save all entries always instead of just the updated ones. Assuming that [this](https://github.com/jruby/dbm) will load when I go "require dbm" I'm willing to give that a shot, but SQLite is the only thing that's performed adequately so far.